evil-collection.el

```diff
@@ -363,6 +363,26 @@ to filter keys on the basis of `evil-collection-key-whitelist' and
                                 err))))))
              (add-hook 'after-load-functions fun t))))))
 
+(cl-defun evil-collection-translate-key-to-state (state
```

Comment:
About the name, what about `evil-collection-forward-to-emacs-state`?

Reply:
I went with `evil-collection-move-bindings-to-emacs-state` instead because yours sounds like it could be an interactive function.